media-sound/pulseaudio-modules-bt/metadata.xml

```diff
@@ -7,4 +7,15 @@
 <maintainer type="person">
 	<email>pacho@gentoo.org</email>
 </maintainer>
+<use>
+	<flag name="fdk">
+	>=0.1.5: pulseaudio-modules-bt use LC-AAC only
```

Comment:
Maybe a bit more detail about what this means?

Reply:
Let me add more details about it on my following commit. My bad for copy/paste from pulseaudio-modules-bt github documentation.

LC-AAC is just an AAC profile for audio codec transmission, there is LC-AAC (Low Complexity AAC) and HE-AAC (High Efficiency AAC) profiles as standards.

Reply:
I just wasn't sure what >=0.1.5 was referring to and didn't know if a user would.

Don't need to worry about explaining AAC etc though. Thank you!

Reply:
No need to make a version reference. Use flag description is only shown for the user visible ebuild that contains such flag.